symbolic-minidump/tests/test_processor.rs

```diff
@@ -12,6 +16,32 @@ fn process_minidump_linux() -> Result<(), Error> {
     Ok(())
 }
 
+#[test]
+fn process_minidump_linux_cfi() -> Result<(), Error> {
+    let buffer = ByteView::open(fixture("linux/mini.dmp"))?;
+    let mut frame_info = FrameInfoMap::new();
+
+    let cfi_records = {
+        let file = BufReader::new(File::open(fixture("linux/crash.sym"))?);
+
+        // Read STACK CFI records starting at line 170
+        file.lines()
```

Comment:
Is there a better way to read from line 170 than skipping here?

Reply:
this seems reasonable to me